Assets/Scripts/Algorithms/Patrolling/CognitiveCoordinated/CognitiveCoordinatedBase.cs

```diff
@@ -11,15 +11,21 @@
 
 namespace Maes.Algorithms.Patrolling
 {
-    public sealed class CognitiveCoordinated : PatrollingAlgorithm
+    public abstract class CognitiveCoordinatedBase : PatrollingAlgorithm
     {
-        public override string AlgorithmName => "Cognitive Coordinated Algorithm";
+        protected const int VirtualStigmergyMessageId = 0;
         private List<Vertex> _currentPath = new();
         private int _pathStep;
         private readonly Dictionary<(int, int), Vertex[]> _pathsCache = new();
 
         protected override bool AllowForeignVertices => true;
```

Comment:
This should only be true in the static coordinator version. Override it there.

Reply:
The algo name?